aviary/data.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+from typing import Callable, Iterator
+
+import numpy as np
+from torch import Tensor
+
+np.random.seed(0)
```

Comment:
I think this might not be best practice? Why do we need to set the seed explicitly here

Reply:
We can also do it inside the class if you prefer?

Reply:
I think setting the seed globally is bad as it significantly reduces the randomness. Here I am not sure why specifying a seed is needed?  

I don't have much experience but I think the recommended approach now is to use a generator and then call the random number generation functions on that. https://numpy.org/doc/stable/reference/random/generator.html

Reply:
I see what you mean about global random state. I'll use a local RNG. Or we could leave it unseeded. The reason I used a seed is to have deterministic shuffling:

https://github.com/CompRhys/aviary/blob/735beec93e37abba46612817a505caa6df82c5f5/aviary/data.py#L37-L39